src/frontend/apps/standalone_site/src/components/Modal/ModalButton.tsx

```diff
@@ -0,0 +1,82 @@
+import { Box, Button, Stack } from 'grommet';
+import { Spinner } from 'lib-components';
+import { defineMessages, useIntl } from 'react-intl';
+import styled from 'styled-components';
+
+const messages = defineMessages({
+  ButtonCancelModal: {
+    defaultMessage: 'Cancel',
+    description: 'Label for cancel button in modal.',
+    id: 'components.Modal.ButtonCancelModal',
+  },
+});
+
+interface ButtonModalProps {
+  flex?: string | number;
+}
+const ButtonModal = styled(Button)<ButtonModalProps>`
+  flex: ${({ flex }) => flex && `${flex}`};
+`;
+
+interface ModalButtonProps {
+  label: string;
+  isDisabled?: boolean;
+  isSubmiting?: boolean;
+  onClickSubmit?: () => void;
+  onClickCancel?: () => void;
+}
+
+/**
+ * @param ModalButtonProps
+ *  - Init onClickCancel will display the cancel button
+ * @returns ModalButton component
```

Comment:
I don't know if it's useful and it seems incomplete.

Reply:
I can see some modals without the cancel:
[Modal without cancel](https://app.abstract.com/projects/a05c2d87-39b7-439e-b7e9-4b1ac6667ce3/branches/a3b41a47-2cc3-47e8-9ee9-05ccb7926eca/commits/latest/files/1d6d9477-aff0-4d93-b3bd-982068b16af3/layers/34C1260D-6ED1-4695-8624-7D34419E6B94?collectionId=87bad8d2-4099-4b2d-b1a9-465e84b00355&collectionLayerId=7e640230-eda7-4d92-919d-f5cdb7536e24)